salt/states/kapacitor.py

```diff
@@ -63,6 +71,7 @@ def task_present(name,
 
     task = __salt__['kapacitor.get_task'](name)
     old_script = task['script'] if task else ''
+    task_dbrps = [{'db': dbrp[0], 'rp': dbrp[1]} for dbrp in (dbrp.split('.') for dbrp in dbrps)]
```

Comment:
How does this change things for people specifying `database` and `retention_policy` and not `dbrps`? It seems like this would change the default behavior.